### ISSUE TITLE
shopp_add_notice() enhancement

### ISSUE THREAD

**Issue by crunnells** [ingenesis/shopp#3295]
Just found this little gem in the code while looking for a way to push non-error messages to the cart. Right now, it just outputs an error to the cart (so it's essentially `shopp_add_error()` but without the option of specifying whether it's a front-end or an admin error), but it would be nice if it would output as a true "notice" instead of an error. This would require adding a new class "ShoppNotice" maybe? Or extending ShoppError to have non-error messages?

Maybe I'm barking up the wrong tree here. Thoughts anyone?

**Comment by jonathandavis** [comment]
There has been a little work on restructuring error handling. Most of it relates to how it is kept in session state for the admin vs front-end.

I think this is part of a larger project. Rather than adding to an encumbered system that isn't designed with a well-established notion of storefront vs admin notices (there isn't any clear distinction), we should look at an ideal design and map how to get there from here. Anyone interested in shepherding that project?

**Comment by crunnells** [comment]
What I'm proposing is simple. Basically, just copying (and simplifying) the error class, and changing it so that it displays customer notices on the front end. Consider this scenario:

![screenshot 2015-05-22 14 29 57](https://cloud.githubusercontent.com/assets/4297811/7781680/677f64de-008f-11e5-9dfe-69957a74ead3.png)

This message is _not_ an error, but it's under an "Errors" heading. This is misleading. We run into the same problem in the cart, where anything passed to `shopp_add_notice()` is construed as an error, even though it shouldn't be (eg: "This item is low on stock, order now!").

**Comment by meredevelopment** [comment]
Agreed this needs some work. How about adding a 'severity' option to [shopp_add_error](https://github.com/ingenesis/shopp/blob/master/api/core.php#L575-L578) which could be used in [storefront.errors](https://github.com/ingenesis/shopp/blob/master/api/theme/storefront.php#L764-L781) to identify the list items differently. Maybe.
